src/core/Akka.Remote.Tests/Serialization/DaemonMsgCreateSerializerSpec.cs

```diff
@@ -34,6 +34,8 @@ protected override void OnReceive(object message)
         public DaemonMsgCreateSerializerSpec()
             : base(@"akka.actor.provider = ""Akka.Remote.RemoteActorRefProvider, Akka.Remote""")
         {
+            Akka.Serialization.Serialization.CurrentTransportInformation = null;
```

Comment:
I'd like to call this fix out. Without this when you run a test runner on the assembly the objects get the system name of the other tests.